.editorconfig

```diff
@@ -0,0 +1,15 @@
+# This file is for unifying the coding style for different editors and IDEs
+# https://editorconfig.org
+
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
1. Is this specifically for the GitHub classroom setup or for the students direct benefit?
2. If it's for the students benefit we need to make a note somewhere of the VS Code extensions they'll need to install

Reply:
For general benefit. You’re right, do we have a list of applications we’re providing in advance, or will it all be dealt with in the prep session?

Reply:
@tanyapowell Given the EditorConfig extension wasn't included in the prep session, should I just remove this?

Reply:
Yes please. We can introduce this from JS onwards